types/queries.go

```diff
@@ -402,6 +402,8 @@ type ContractInfoResponse struct {
 	Pinned bool   `json:"pinned"`
 	// Set if the contract is IBC enabled
 	IBCPort string `json:"ibc_port,omitempty"`
+	// Set if the contract is Eureka enabled
+	EurekaPort string `json:"eureka_port,omitempty"`
```

Comment:
So this also needs to be added in cosmwasm-std.

Reply:
Right. Thanks for pointing this out